docs/deployments/apis.md

```diff
@@ -1,16 +1,37 @@
 # APIs
 
-Serve models at scale.
+Deploy your model as an API at scale. 
+
+A python interface below can be implemented to deploy most models with Cortex. Specify a python `inference` file that describes how to load your model and use it to make predictions. Cortex uses the `inference` file to deploy multiple replicas that can serve your model as an API. Deployment parameters such as minimum replica count, maximum replica count, enabling prediction monitoring can be configured using yaml.
+
+Besides providing a python interface, Cortex can serve the following exported model formats:
+- [TensorFlow saved model](./tensorflow/api.md)
+- [ONNX](./onnx/api.md)
+
+
+## Inference
+Model initialization and other setup such as downloading vocabulary or initializing a tokenizer can be defined in the `init` function. The `predict` function is responsible for making an inference on a request and returning a prediction. The preprocessing of a request and postprocessing of model output can be done in the predict function.
+
+```python
+import ...
+
+def init(metadata):
+  # run more initializations here that may require metadata that you define in your yaml
```

Comment:
Does this "more" refer to global initializations? Should we add a comment above illustrating that?